### PR TITLE
Add documentation with using StackHPC helm chart instead

### PR DESCRIPTION
**Pull request by DavidFair**
Replaces the old way of deploying with a new StackHPC deployment method that simplifies things from a user POV

### CODE REVIEW

File: source/ClusterAPI/Setup.rst [hunk 1]
```diff
@@ -55,192 +45,132 @@ cloud Ubuntu image, not the stripped down CAPI image designed for nodes.
...
+            auth_url: https://openstack.stfc.ac.uk:5000/v3
+            application_credential_id: ""
+            application_credential_secret: ""
+        region_name: "RegionOne"
+        interface: "public"
+        identity_api_version: 3
+        auth_type: "v3applicationcredential"
 
-To the floating IP pre-allocated in the project:
+- Add the UUID of the project to create the cluster in. This can be found `here <https://openstack.stfc.ac.uk/identity/>`_.
```
Comment (meoflynn) [inline source/ClusterAPI/Setup.rst]: Where does this project uuid need to be added?

File: source/ClusterAPI/Setup.rst [hunk 2]
```diff
@@ -12,16 +12,6 @@ Setup a new cluster using Cluster API
...
-Deployment Account
-------------------
-
-The deployment is tied to the account which created it. This includes their
-credentials being encoded as an opaque secret within the cluster for the cluster API openstack component.
-
-For production workloads, and long-lived clusters with multiple accounts a service account it recommended.
-This means the cluster isn't tied to the lifetime of a single user's account or password. A service
-account can be requested through our :ref:`contact_details`.
-
```
Comment (meoflynn) [inline source/ClusterAPI/Setup.rst:24]: I think we should keep the original CAPI docs alongside the new docs for now - maybe keep the original docs in a separate page?

File: source/ClusterAPI/Setup.rst [hunk 3]
```diff
@@ -33,7 +23,7 @@ to quickly gain access and perform recovery or upgrades as required.
 Account Security
 ----------------
 
-A `clouds.yaml` file with the user's password is also required. This file should be removed or restricted
+A `clouds.yaml` file with the application credentials is also required. This file should be removed or restricted
```
Comment (meoflynn) [inline source/ClusterAPI/Setup.rst]: Should we also mention for application credentials when they are created they should **never** be unrestricted application credentials?

File: source/ClusterAPI/Setup.rst [hunk 4]
```diff
@@ -55,192 +45,132 @@ cloud Ubuntu image, not the stripped down CAPI image designed for nodes.
...
-- Create the cluster by applying the generated cluster definition:
+- Place this file in the scd-capi-values directory you cloned earlier.
 
-.. code-block:: bash
+Creating the cluster
+====================
 
-    kubectl apply -f $CLUSTER_NAME.yaml
+Configuring the cluster
+-----------------------
```
Comment (meoflynn) [inline source/ClusterAPI/Setup.rst:171]: Maybe we could add a summary of what each YAML file is used for by the Helm chart here?

File: source/ClusterAPI/Setup.rst (hunk 2 above)
Comment (DavidFair) [reply source/ClusterAPI/Setup.rst:24]: I would argue not, it's in source control and I'm not keen to support this method at all going forwards

I'm happy to talk about it offline though if you feel strongly :slightly_smiling_face:

**Review by meoflynn — APPROVED** [review]

**Pull request merged**